pkgs/top-level/all-packages.nix

```
@@ -26753,6 +26755,8 @@ in
   dart_stable = dart.override { version = "2.7.2"; };
   dart_dev = dart.override    { version = "2.9.0-4.0.dev"; };
 
+  buildDartPackage = callPackage ../build-support/dart/default.nix { };
```

default.nix is the... well... default
```suggestion
  buildDartPackage = callPackage ../build-support/dart { };
```